lib/search_files.js

```diff
@@ -0,0 +1,291 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var glob = require('glob');
+
+
+function isWindows() {
+  return process.platform === 'win32';
+}
+
+function isFunction(v) {
+  return (typeof v === 'function');
+}
+
+function isObject(v) {
```

Comment:
just a thought, adding lodash to do the `typeof` checks will add ~40-50 ms load time to this lib. doesn't seem like much but 100ms is noticeable to users, and it might push the load time into that range. on the other hand it does make the code a bit cleaner. 

maybe consider something like https://www.npmjs.com/package/kind-of instead? it's very fast and battle tested